backend/views/syson-tree-explorer-view/src/main/java/org/eclipse/syson/tree/explorer/view/SysONExplorerTreeDescriptionProvider.java

```diff
@@ -88,4 +92,12 @@ private TreeItemLabelFragmentDescription getDefaultLabelFragmentDescription() {
                 .labelExpression("aql:self.getLabel()")
                 .build();
     }
+
+    private List<TreeItemContextMenuEntry> getContextMenuEntries() {
+        var expandAllMenuEntry = new TreeBuilders().newCustomTreeItemContextMenuEntry()
+                .contributionId("expandAll")
```

Comment:
"expandAll" should be extracted as a constant and reused in `SysONExplorerTests#getContextMenuOfModelAndLibraryDirectories` and in `SysONExplorerTreeItemContextMenuEntryProvider#getTreeItemContextMenuEntries `